peepholelib/classifier/tgmm.py

```diff
@@ -60,4 +60,17 @@ def classifier_probabilities(self, **kwargs):
         data, _ = self.parser(data = batch, **self.parser_kwargs)
         probs = torch.tensor(self._classifier.predict_proba(data), dtype=data.dtype)
         return probs  
+    
+    def save(self, **kwargs):
+        self.file_path = self.path/('GMM.' + f'cv_dim={self._classifier.model_.config.num_features}' + '.' + f'num_cluster={self.nl_class}')
+        self.file_path.mkdir(parents=True, exist_ok=True)
+        self._classifier.save(self.file_path)
+        super().save()
+
+    def load(self, **kwargs):
+        self.file_path = kwargs['file_path']
```

Comment:
reconstruct `file_path` from `nl_class` and `nl_features`, the same path as in the  `save()`.
Do not save `file_path` into `self.`